packages/client/src/Components/Mapping/components/Adjudication/index.jsx

```diff
@@ -29,17 +30,25 @@ const Adjudication = ({ store, onStoreUpdate, callbackFunction }) => {
 
   newStore.formType = 'adjudication'
 
-  // HACK: This is just for the current wargame, this will need to be replaced with a non text-based comparison
+  const conditions = allPlatforms[0].conditions
+  const getDisabledCondition = conditions[conditions.length - 1]
 
-  const checkIfActive = () => {
-    const activeArray = ['working', 'full-capability']
-    return activeArray.includes(_.kebabCase(markerCondition))
+  const checkIfActive = (val) => {
+    return val !== getDisabledCondition
+  }
+
+  const checkIfDeploying = () => {
+    return markerStatus === 'Deploy'
   }
```

Comment:
One subtle change here. You were close, good guess, but I'd like us to avoid the dependency on the string.

For `platformTypes` some states include a `deploying` flag:
![image](https://user-images.githubusercontent.com/1108513/73877978-9c4c0500-4851-11ea-972b-574b9d7e9ca0.png)

We should test against that flag, please.

Reply:
Done.